aries-site/src/pages/templates/wizard.mdx

```diff
@@ -28,7 +29,7 @@ It is important to give the user an indication of where they are within the form
 
 ### Validation
 
-Validation should occur within each step of the wizard. This can happen `onBlur` of each input or when the user clicks the button that would advance them to the next step. If there are any errors with the inputs in the current step, the user should not be able to advance.
+Validation should occur within each step of the wizard. If there are any errors with the inputs in the current step, the user should not be able to advance and errors should be displayed. Error display should follow conventions established by [Form](/templates/forms).
```

Comment:
Wonder if we should reference and provide a link to the example below?

Reply:
Yes I think that would be helpful

Reply:
Implemented.